easypy/concurrency.py

```diff
@@ -377,22 +390,31 @@ def kill(wait=False):
 
 def concurrent_find(func, params, **kw):
     timeout = kw.pop("concurrent_timeout", None)
-    with async(func, list(params), **kw) as futures:
```

Comment:
is this refactoring actually needed? `async` takes a `workers` parameter, and if you break out in the middle it should terminate the executor and the remaining tasks...